cmd/ncproxy/hcn_networking_test.go

```diff
@@ -964,12 +970,18 @@ func TestAddEndpoint_NoError(t *testing.T) {
 			networkCreateFunc: createTestDualStackNATNetwork,
 		},
 	}
+	for i, n := 0, len(tests); i < n; i++ {
+		t := tests[i]
+		t.name += " - host attach"
+		t.attachToHost = true
+		tests = append(tests, t)
+	}
```

Comment:
why not just add the new tests directly in the test slice above?

Reply:
I was thinking there would be multiple places that id need to update/add more test cases, but ill do that and clean it up